### ISSUE TITLE
Player does not allow volume to be adjusted

### ISSUE THREAD

**Issue by AndrewX192** [UWave/uwave.fm#2]
Some people do not have volume controls on their computer and are unable to adjust the volume without using the player volume control.

**Comment by hasit** [comment]
Could we use a media player that would allow volume control? Does anyone know of one?

**Comment by thefinn93** [comment]
I'm fairly certain the the HTML5 player allows javascript to set the volume, it shouldn't be that hard to implement

**Comment by AndrewX192** [comment]
Yes, the volume can be set (at least in Firefox 36) with the following javascript: 
```javascript
$('#player')[0].volume = .7
```
